server/lib/two-factor-authentication/lib.ts

```diff
@@ -183,14 +183,13 @@ async function shouldEnforceForAccount(req, account?: typeof models.Collective):
 }
 
 /**
- * Enforce 2FA if the remote user is an admin of `account` (or root) and this account has
- * the `REQUIRE_2FA_FOR_ADMINS policy` set on itself or its parent.
+ * Enforce 2FA if enabled on `account` and this account has the `REQUIRE_2FA_FOR_ADMINS policy` set on itself or its parent.
  *
- * Otherwise, this function will still check for 2FA if it's enabled on the user account.
+ * Otherwise, this function will still check for 2FA for root users or if it's already enabled on the user account.
  *
  * @returns true if 2FA was validated, false if not required
  */
-async function enforceForAccountAdmins(
+async function enforceForAccount(
```

Comment:
Renamed this one for clarity as we were not really checking whether the user was an admin of `account`. Introduced `enforceForAccountsUserIsAdminOf` instead to achieve this.